website/docs/components/form/layout/partials/specifications/anatomy.md

```diff
@@ -0,0 +1,15 @@
+## Anatomy
+
+<!-- image then table -->
+![Anatomy of formLayout](/assets/components/alert/form-layout-anatomy-inline.png)
```

Comment:
@LilithJames-HDS Would you be able to add an image and the anatomy info for this section?